src/menu.test.interactions.ts

```diff
@@ -1412,11 +1412,10 @@ it('opens its sub-Menus when its sub-Menu targets are clicked', async () => {
     ),
   );
 
-  // Replaces the usual `await aTimeout(0)`. It's not clear why. But opening Menus
-  // in tests sometimes takes more than a tick when sub-Menus are present.
-  await waitUntil(() => {
-    return defaultSlots[0]?.checkVisibility();
-  });
+  // It's not clear why. But opening and positioning Menus in tests, particularly in
+  // CI, sometimes takes more than a tick when sub-Menus are present. 25 milliseconds
+  // is unfortunate. But it gives the browser plenty of time to do its thing.
```

Comment:
Happy to explain offline what I think is going on. But it's just a suspicion, and I wasn't able to verify my suspicion with a debugging screenshot due to [this](https://github.com/CrowdStrike/glide-core/pull/978/files#diff-24d782e02f46a1eb18aed719d1c73354c23a5702d6e324a6163f6d90a6965cadR68-R71). So I figured a general comment would suffice.

Reply:
By the way, I rebuilt this PR around 10 times without any failures. So I have high confidence it's a timing issue and that 25 milliseconds is sufficient. 

That's not to say over flaky Menu tests won't pop up over the next couple days. But the flakiness we've seen so far should be fixed.